app/components/markdown/index.js

```diff
@@ -303,9 +307,38 @@ export default class Markdown extends PureComponent {
         );
     }
 
+    renderEditedIndicator = ({context}) => {
+        let spacer = '';
+        let opacity = Platform.select({ios: 0.7, android: 1});
+        if (context[0] === 'paragraph') {
+            spacer = ' ';
+            opacity = Platform.select({ios: 0.5, android: 0.6});
+        }
+        const styles = [style.editedIndicatorText, {opacity}];
+        if (Platform.OS === 'ios') {
+            styles.push(this.computeTextStyle(this.props.baseTextStyle, context));
+        }
+        return (<Text style={styles}>
```

Comment:
Small style thing, we usually have the brackets on a separate line when the component spans multiple lines like
```
return (
    <Text
     />
);
```